src/optics/lib/Convex/CardanoApi/Lenses.hs

```diff
@@ -468,15 +459,6 @@ _TxOutDatumHash = prism' from to
   from :: C.Hash C.ScriptData -> TxOutDatum ctx era
   from h = C.TxOutDatumHash C.alonzoBasedEra h
 
-_TxOutDatumInTx :: forall era. (C.IsAlonzoBasedEra era) => Prism' (TxOutDatum CtxTx era) C.HashableScriptData
```

Comment:
Like I mentionned above, we should bring this back, but renamed to `TxOutSupplementalDatum`.

Reply:
ah nice, thanks. For some reason I thought this was completely gone.